lib/transport/JsonRpcTransport/errors/JsonRpcError.js

```diff
@@ -0,0 +1,63 @@
+class JsonRpcError extends Error {
```

Comment:
We have base DAPIClientError where we are doing all necessary stuff in constructor